src/main/scala/tile/BaseTile.scala

```diff
@@ -117,6 +117,7 @@ trait HasNonDiplomaticTileParameters {
       Option.when(tileParams.core.useBitManip)(Seq("zbs")) ++
       Option.when(tileParams.core.useCryptoNIST)(Seq("zknd", "zkne", "zknh")) ++
       Option.when(tileParams.core.useCryptoSM)(Seq("zksed", "zksh")) ++
+      Option.when(tileParams.core.useConditionalZero)(Seq("zicond")) ++
```

Comment:
According to the isa spec, zicond should be put before zicsr and after zicntr.

>The first letter following the “Z” conventionally indicates the most closely related alphabetical
extension category, IMAFDQCVH. For the “Zam” extension for misaligned atomics, for exam-
ple, the letter “a” indicates the extension is related to the “A” standard extension. If multiple
"Z" extensions are named, they should be ordered first by category, then alphabetically within a
category—for example, “Zicsr Zifencei Zam”.